.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+# https://pre-commit.com/
+repos:
+-   repo: https://github.com/thlorenz/doctoc
+    rev: v2.2.0
+    hooks:
+    -   id: doctoc
+        args: [--update-only]
```

Comment:
This makes `doctoc` not force itself on everything, while it can be enabled by adding the following anywhere in a file:

```html
<!-- START doctoc -->
<!-- END doctoc -->
```

However it looks like all files were doctoced before I enabled it, and I don't know whether non-doctoced are desirable.